dev/com.ibm.ws.persistence_fat/publish/servers/com.ibm.ws.persistence.consumer.jpa30/server.xml

```diff
@@ -0,0 +1,35 @@
+<server>
+    <include location="../fatTestPorts.xml" />
+
+    <featureManager>
+        <feature>servlet-5.0</feature>
+        <feature>jpa-3.0</feature>
+        <feature>ejbLite-4.0</feature>
```

Comment:
Must change "ejbLite-4.0" to "enterpriseBeansLite-4.0"

Reply:
Addressed!